docs/react/guides/important-defaults.md

```diff
@@ -20,7 +20,7 @@ Out of the box, TanStack Query is configured with **aggressive but sane** defaul
 - Query results that have no more active instances of `useQuery`, `useInfiniteQuery` or query observers are labeled as "inactive" and remain in the cache in case they are used again at a later time.
 - By default, "inactive" queries are garbage collected after **5 minutes**.
 
-  > To change this, you can alter the default `gcTime` for queries to something other than `1000 * 60 * 5` milliseconds.
+  > To change this, you can alter the default `gcTime` for queries to something other than `1000 * 60 * 5` milliseconds. Avoid setting `gcTime` to `0` as that can cause a hydration error.
```

Comment:
I would rather put this as a separate section in `Tips, Tricks and Caveats` in the [SSR docs page](https://tanstack.com/query/latest/docs/react/guides/ssr#tips-tricks-and-caveats), because it's strictly SSR related.

We already have a paragraph there explaining why `gcTime` defaults to `Infinity` [on the server](https://tanstack.com/query/latest/docs/react/guides/ssr#high-memory-consumption-on-server), so adding another one why setting it to zero is not a good idea would be good.

Reply:
Updated 👍